src/sentry/api/endpoints/integrations/sentry_apps/index.py

```diff
@@ -128,7 +128,8 @@ def post(self, request: Request, organization) -> Response:
                     overview=data["overview"],
                     allowed_origins=data["allowedOrigins"],
                     popularity=data["popularity"],
-                ).run(user=request.user, request=request)
+                ).run(user=request.user, request=request, skip_default_auth_token=True)
+                # We want to stop creating the default auth token for new apps and installations through the API
```

Comment:
do you know where we're still creating the default auth token?

Reply:
Through the code level, and other flows, we depend on this a lot. For example, a lot of the SCIM code depends on an installation coming with an auth code out of the box. I think that area would be a bigger refactor unfortunately, because it seems like the assumption is an installation/app comes with a token for the domain

Reply:
oh i see. this is where all the SCIM tests were failing then